styles/palette.css

```diff
@@ -15,6 +17,8 @@
   --stanford-stone-dark-rgb: 84, 73, 72;
   --stanford-illuminating-dark-rgb: 254, 197, 29;
   --stanford-fog-light: #f4f4f4;
+  --stanford-sky-dark: #016895;
```

Comment:
Can we omit these since they aren't used in the stylesheet anywhere?

Reply:
Moved it to the `<style>` in the heads of the templates.